File: kids_with_candies_low.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        result = []
        max_candy = max(candies)
        for i in candies:
            result.append(extraCandies + i >= max_candy)
        return result

File: kids_with_candies_low.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int max_candy = 0;
        for(int i : candies){
            max_candy = max(i,max_candy);
        }
        vector<bool> result(candies.size());
        for(int i = 0;i < candies.size(); i++){
            result[i] = (candies[i] + extraCandies >= max_candy);
        }
        return result;
    }
};
